### ISSUE TITLE
@filledout/yup ERESOLVE unable to resolve dependency tree

### ISSUE THREAD

**Issue by frkam** [filledout/filledout#1]
I installed everything according to the documentation and got the following error on installing @filledout/yup

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: xxx@0.0.0
npm ERR! Found: @filledout/core@0.0.9
npm ERR! node_modules/@filledout/core
npm ERR!   @filledout/core@"^0.0.9" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @filledout/core@"0.0.8" from @filledout/yup@0.0.4
npm ERR! node_modules/@filledout/yup
npm ERR!   @filledout/yup@"0.0.4" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/xxx/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/xxx/.npm/_logs/2023-03-18T21_26_42_003Z-debug-0.log
```

@filledout/core version: `0.0.9`
@filledout/react version: `0.0.8`

**Comment by hyze2d** [comment]
Can you provide a repro?

**Comment by frkam** [comment]
> Can you provide a repro?

repro: https://github.com/frkam/filledout-eresolve-repro
this repo contain main dependencies. if you will try install @filledout/yup you will get error that i wrote about.

**Comment by ifeelBALANCED** [comment]
@hyze2d  @frkam fixed in @filledout/yup@1.0.0-alpha.1 ?
![image](https://github.com/filledout/filledout/assets/55098650/be296f2f-dc7f-4d27-9226-03ec06581ec1)
Everything file

**Comment by hyze2d** [comment]
As said above could be possibly fixed in alpha but keep in mind that it has api changes which are not documented yet. Docs update planned later this week
